calcom/cal.com
# fix: paypal save event type and payment

## What does this PR do?

- Fixes capture return waiting for body when it's not there anymore.
- Fixes save on event type price | currency on event type for stripe and paypal.


## How should this be tested?

- Are there environment variables that should be set? No.
- What are the minimal test data to have? Paypal app in an event type.
- What is expected (happy path) to have (input and output)? 
- Any other important info that could help to test that PR

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
